lib/Sinek.js

```diff
@@ -1,33 +1,12 @@
 "use strict";
 
-const util = require("util");
-
-const Kafka = require("./kafka/Kafka.js");
-const Drainer = require("./kafka/Drainer.js");
-const Publisher = require("./kafka/Publisher.js");
-const PartitionDrainer = require("./kafka/PartitionDrainer.js");
-
-const Consumer = require("./connect/Consumer.js");
-const Producer = require("./connect/Producer.js");
-
-const NConsumer = require("./librdkafka/NConsumer.js");
-const NProducer = require("./librdkafka/NProducer.js");
-
 const JSConsumer = require("./kafkajs/JSConsumer.js");
 const JSProducer = require("./kafkajs/JSProducer.js");
 
 const Health = require("./shared/Health.js");
 const Analytics = require("./shared/Analytics.js");
 
 module.exports = {
-  Kafka: util.deprecate(Kafka, "Kafka is deprecated, please use 'NConsumer' if possible."),
```

Comment:
we should definitely mention this clearly in the change log, we removed the depracted client versions and explicity removed the native client from the whole lib